frontend/app/components/ItemCategory.tsx

```diff
@@ -0,0 +1,132 @@
+'use client';
+import React, { useState, useEffect } from "react";
+import Image from "next/image";
+
+export default function ItemCategory() {
+
+    // --------------------------------------------------------
+
+    const testCategoryItems = [
+        {
+            toolID: 1,
+            name: "Hunting Axe",
+            lender: "Jarl Balgruff",
+            lenderRating: "4.9",
+            location: "Calgary, AB",
+            image: "/axe.jpg"
+        },
+        {
+            toolID: 2,
+            name: "Woodcutting Axe",
+            lender: "Jarl Balgruff",
+            lenderRating: "4.9",
+            location: "Calgary, AB",
+            image: "/axe.jpg"
+        },
+        {
+            toolID: 3,
+            name: "Hatchet",
+            lender: "Jarl Balgruff",
+            lenderRating: "4.9",
+            location: "Calgary, AB",
+            image: "/axe.jpg"
+        },
+        {
+            toolID: 4,
+            name: "Fire Axe",
+            lender: "Jarl Balgruff",
+            lenderRating: "4.9",
+            location: "Calgary, AB",
+            image: "/axe.jpg"
+        },
+        {
+            toolID: 5,
+            name: "Carpenter Axe",
+            lender: "Jarl Balgruff",
+            lenderRating: "4.9",
+            location: "Calgary, AB",
+            image: "/axe.jpg"
+        },
+        {
+            toolID: 6,
+            name: "Tomahawk Axe",
+            lender: "Jarl Balgruff",
+            lenderRating: "4.9",
+            location: "Calgary, AB",
+            image: "/axe.jpg"
+        },
+        {
+            toolID: 7,
+            name: "Felling Axe",
+            lender: "Jarl Balgruff",
+            lenderRating: "4.9",
+            location: "Calgary, AB",
+            image: "/axe.jpg"
+        },
+        {
+            toolID: 8,
+            name: "Hunting Axe",
+            lender: "Jarl Balgruff",
+            lenderRating: "4.9",
+            location: "Calgary, AB",
+            image: "/axe.jpg"
+        }
+    ]
+
+    const testCategoryName = "Axes"
+
+    // --------------------------------------------------------
+
+    interface ToolItem {
+        toolID: any;
+        name: any;
+        lender: any;
+        lenderRating: any;
+        location: any;
+        image: any;
+    }
```

Comment:
I have an interface folder where you can put your interfaces in. This is so that other files can access this interface without conflict.
```suggestion

```